frontend/src/App.jsx

```diff
@@ -67,8 +67,8 @@ function App() {
            */}
         </Routes>
       </main>
-      <Header />
       <Footer />
+      <Header />
```

Comment:
Euh.... Why????

Reply:
to display menu burger in front of the footer in mobile version.